.github/workflows/ci.yml

```diff
@@ -268,6 +268,7 @@ jobs:
           ulimit -c unlimited -S
           timeout --signal=SIGABRT 60m ./tst/webrtc_client_test
   sample-check:
+    if: github.repository == 'awslabs/amazon-kinesis-video-streams-webrtc-sdk-c'
```

Comment:
Why are we doing this? We can modify the script instead: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/2dff44fea83a79336fda676ba2801d0d77417cd1/scripts/check-sample.sh#L3. Replace with checking for `AWS_REGION` or `AWS_ROLE_TO_ASSUME` or `AWS_ROLE_SESSION_NAME`?

Reply:
We are doing this because the job fails way before that on the "Configure AWS Credentials" step for forked branches. Even if we move that step right before the script execution, it would fail on this step because GitHub does not allow accessing secrets for forked branches at all

Reply:
Ok. Makes sense. How does setting repo work in itself? How does this detect a forked branch?

Reply:
I referred [this](https://github.community/t/stop-github-actions-running-on-a-fork/17965/2)